hironx_ros_bridge/rqt_plugin.xml

```diff
@@ -0,0 +1,22 @@
+<library path="src">
+  <class name="HiroNXDashboard" type="hironx_ros_bridge.hironx_dashboard.HiroNXDashboard" base_class_type="rqt_gui_py::Plugin">
+    <description>
+      A Python GUI plugin for displaying a dashboard that displays and interacts with low level hrpsys states.
+    </description>
+    <qtgui>
+      <group>
+        <label>Robot</label>
+        <icon type="theme">folder</icon>
+        <statustip>Plugins related to specific robots.</statustip>
+      </group>
+      <group>
```

Comment:
did you run rqt --force-discover (
http://wiki.ros.org/rqt/Tutorials/Create%20your%20new%20rqt%20plugin#Install_.26_Run_your_plugin
)

--
◉ Kei Okada


On Wed, Feb 4, 2015 at 2:03 AM, Isaac I.Y. Saito <notifications@github.com>
wrote:

> In hironx_ros_bridge/rqt_plugin.xml
> <https://github.com/start-jsk/rtmros_hironx/pull/317#discussion_r24019750>
> :
>
> > @@ -0,0 +1,22 @@
> > +<library path="src">
> > +  <class name="HiroNXDashboard" type="hironx_ros_bridge.hironx_dashboard.HiroNXDashboard" base_class_type="rqt_gui_py::Plugin">
> > +    <description>
> > +      A Python GUI plugin for displaying a dashboard that displays and interacts with low level hrpsys states.
> > +    </description>
> > +    <qtgui>
> > +      <group>
> > +        <label>Robot</label>
> > +        <icon type="theme">folder</icon>
> > +        <statustip>Plugins related to specific robots.</statustip>
> > +      </group>
> > +      <group>
>
> I haven't been able to see this rqt plugin listed on my rqt window. I
> suspect having multiple group tags may not work, but does this work for
> you?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/start-jsk/rtmros_hironx/pull/317/files#r24019750>.
>